src/send/SendSelectRecipient.tsx

```diff
@@ -201,6 +202,14 @@ function SendSelectRecipient({ route }: Props) {
   const { recipientVerificationStatus, recipient, setSelectedRecipient, unsetSelectedRecipient } =
     useFetchRecipientVerificationStatus()
 
+  // TODO(satish/joe): update condition to not show this if the address
```

Comment:
Does this have an associated ticket?

Reply:
ACT-979 (same as this ticket), will do a fast follow once #4586 is merged